src/client/activation/extensionSurvey.ts

```diff
@@ -86,6 +90,11 @@ export class ExtensionSurveyPrompt implements IExtensionSingleActivationService
     }
 
     private launchSurvey() {
-        this.browserService.launch('https://aka.ms/AA5rjx5');
+        const extensionVersion = encodeURIComponent(this.appEnvironment.packageJson.version);
+        const vscodeVersion = encodeURIComponent(vscode.version);
+        const machineId = encodeURIComponent(this.appEnvironment.machineId);
+        const platform = encodeURIComponent(this.platformService.osType);
+        const url = `https://aka.ms/AA5rjx5?o=${platform}&v=${vscodeVersion}&e=${extensionVersion}&m=${machineId}`;
```

Comment:
I recommend using [node's "querystring" module](https://nodejs.org/api/querystring.html):

```suggestion
        const query = querystring.stringify({
            o: encodeURIComponent(this.platformService.osType), // platform
            v: encodeURIComponent(vscode.version),
            e: encodeURIComponent(this.appEnvironment.packageJson.version), // extension version
            m: encodeURIComponent(this.appEnvironment.machineId)
        });        
        const url = `https://aka.ms/AA5rjx5?${query}`;
```

Reply:
Didn't know this, thanks. Looks a lot better